tests/e2e/specs/editor/media/insertMovVideo.js

```diff
@@ -37,8 +38,13 @@ describe('Inserting .mov from dialog', () => {
     await page.waitForSelector(MODAL, {
       visible: true,
     });
-
-    const fileName = await uploadFile('small-video.mov', false);
+    const testMediaPath = join(
+      __dirname,
+      '../../..',
+      'assets',
+      'small-video.mov'
+    );
+    const fileName = await uploadFile(testMediaPath, false);
```

Comment:
Having to use `join` and `__dirname` everytime to upload a file adds a lot of repetition. It was much more DRY before.

Reply:
To fix this, I would have to move the asset files into the e2e package, I am creating here. Should I do this?

Reply:
https://github.com/google/web-stories-wp/pull/6275/commits/6dc648c63ebf123a98300fe38bede9013caa014e

Reply:
Fixed with a work around.

Reply:
Oh, I was just about to reply to your comment, saying  that I now see why you did this and that it probably is fine (as long as the ESLint config is adequately updated).

It's an interesting workaround though 👀 I think we can use that for now